sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -385,7 +385,9 @@ case class LoadDataCommand(
         val hadoopConf = sparkSession.sessionState.newHadoopConf()
         val srcPath = new Path(hdfsUri)
         val fs = srcPath.getFileSystem(hadoopConf)
-        if (!fs.exists(srcPath)) {
+        // Check if the path exists or there are matched paths if it's a path with wildcard.
+        // For HDFS path, we support wildcard in directory name and file name.
+        if (null == fs.globStatus(srcPath) || fs.globStatus(srcPath).isEmpty) {
```

Comment:
@HyukjinKwon @dongjoon-hyun  Is it possible to use `fs.globStatus` for both local path and hdfs path?

Reply:
Up to my knowledge, it's fine to use `fs.globStatus` for both hdfs and local path. I think that Hadoop's interface should guarantee it at least.

Reply:
@wzhfy @HyukjinKwon @dongjoon-hyun  i verified the scenario by updating the code by using fs.globStatus() API for both local and hdfs path. for local path its working fine

Reply:
I will update the PR  as such  we can use fs.globStatus() API  in both local and hdfs file path scenarios. Thanks.